flink-connector/src/main/java/org/apache/flink/connector/delta/sink/DeltaSink.java

```diff
@@ -145,15 +157,50 @@
      * @param conf     Hadoop's conf object that will be used for creating instances of
      *                 {@link io.delta.standalone.DeltaLog} and will be also passed to the
      *                 {@link ParquetRowDataBuilder} to create {@link ParquetWriterFactory}
+     * @param rowType  Flink's logical type to indicate the structure of the events in the stream
      * @return builder for the DeltaSink
      */
     public static DeltaSinkBuilder<RowData> forDeltaFormat(
-            final Path basePath,
-            final Configuration conf
+        final Path basePath,
+        final Configuration conf,
+        final RowType rowType
+    ) {
+        conf.set("parquet.compression", "SNAPPY");
+        ParquetWriterFactory<RowData> writerFactory =
+            ParquetRowDataBuilder.createWriterFactory(rowType, conf, true);
+
+        return new DeltaSinkBuilder.DefaultDeltaFormatBuilder<>(
+            basePath,
+            conf,
+            writerFactory
+        );
+    }
+
+    /**
+     * Convenience method for creating {@link DeltaSink}.
+     * Developer must ensure that writerFactory is configured to use SNAPPY compression codec.
```

Comment:
just so you know, when we generate javadocs, newlines are removed and only `<p>` actually provides a new line in the output. So
```
Convenience method for creating {@link DeltaSink}.
Developer must ensure that writerFactory is configured to use SNAPPY compression codec.
```

just becomes

```
Convenience method for creating {@link DeltaSink}. Developer must ensure that writerFactory is configured to use SNAPPY compression codec.
```